modules/dataproc_provisioner/variables.tf

```diff
@@ -84,6 +84,13 @@ variable "encryption_key_name" {
   default     = ""
 }
 
+variable "extra_properties"{
```

Comment:
can you run terraform fmt on all files?

If you use vs code theres a plugin with auto format that  I use: https://marketplace.visualstudio.com/items?itemName=mauve.terraform